L74-java-channel-creds.md

```diff
@@ -0,0 +1,485 @@
+Java: Channel and Server Credentials
+----
+* Author(s): [Eric Anderson](https://github.com/ejona86)
+* Approver: sanjaypujare
+* Status: In Review {Draft, In Review, Ready for Implementation, Implemented}
+* Implemented in: <language, ...>
+* Last updated: 2020-06-17
+* Discussion at: https://groups.google.com/g/grpc-io/c/GNU9svnDsQs/m/kG6XsUy9BwAJ
+
+## Abstract
+
+Develop an API that separates transport credential construction from
+channel/server construction.
+
+This is to allow:
+
+*   implementing new transport credentials without creating a new
+    channel/server type, which allows users of the API to separate concerns
+*   providing stable APIs for advanced credential configuration like mutual
+    TLS, without polluting the general-purpose ManagedChannelBuilder API with
+    many overloads that don't apply to all transports
+*   composition of credentials, like necessary for xDS where a fallback
+    credential needs to be provided by the user
+*   aligning with C core and Go implementations
+
+_The design focuses on channel credentials, since they are the most complex and
+most discussed._ Server credentials will mostly just mirror the channel
+credentials; they suffer from most of the same problems but at a smaller
+scale.
```

Comment:
There is actually a section on Server credentials but this sentence gives the impression server credentials are not covered here. How about rephrasing and moving to the server creds section?

Reply:
I chose to just delete this paragraph.